app/assets/stylesheets/agents.scss

```diff
@@ -20,8 +20,17 @@
   overflow: hidden;
 }
 
-.agent-card {
+.three-col {
+  width: calc(33% - #{$cards-gutter});
+  font-size: 120%;
+  line-height: 2.7em !important;
```

Comment:
OK, onko tuo **!important** - kohta oikeasti pakollinen tuossa koodissa? Jos ei niin suosittelen vahvasti että se poistetaan. 

(Tässä hyvä Stack Overflow asiaa koskien: http://stackoverflow.com/questions/9245353/what-does-important-in-css-mean)